mainwindow.cpp

```diff
@@ -465,7 +466,7 @@ bool MainWindow::eventFilter(QObject *watched, QEvent *event)
     QWidget* widged = (QWidget*) watched;
     switch (event->type())
     {
-    case QEvent::StyleAnimationUpdate:
+    case 1://QEvent::StyleAnimationUpdate:
```

Comment:
I put case 1: just to make it compile. What we have to do is replace QEvent::StyleAnimationUpdate: with something defined old version of Qt that Solar want. I think you'd be more suited than me to correct this as you can reproduce the 30% CPU issue on your laptop.